src/jira/client/jira-client.ts

```diff
@@ -223,8 +223,14 @@ export const getJiraClient = async (
 							repositoryId
 						}
 					}),
-				update: async (data, options?: JiraSubmitOptions) => {
+				update: async (data, gitHubAppId?: number, options?: JiraSubmitOptions) => {
 					dedupIssueKeys(data);
+					const subscription = await Subscription.getSingleInstallation(
```

Comment:
Am I missing something? Why moving the trunk (`await Subscription.getSingleInstallation()` and `await subscription?.update` ) out of the `if{` ?

Reply:
`await Subscription.getSingleInstallation` needs to move so that the subscription is defined outside of the conditional and therefore accessible as an arg for `batchedBulkUpdate`. But the call to subscription.update didn't need to move. Returned that to the conditional.